packages/cli/data/insults.txt

```diff
@@ -0,0 +1,132 @@
+Mirrors can’t talk, lucky for you they can’t laugh either.
```

Comment:
Just want to make sure that these are CC0 licensed. Maybe add a license file in DATA to confirm. And if it's CC-SA, then of course reference where they came from.

Reply:
The first item is alice in wonderland, which has fallen out of copyright.  I added a link to project gutenberg where I downloaded this.  
I wasn't even using the insults, so I've removed them from the changeset.  

You may be interested to know that Alice In Wonderland scores quite badly, often in the 70-80% range, usually because it is deemed incoherent, but sometimes for being insulting.  I'm sure Lewis Carol would not find that amusing:-)